zcash_client_backend/src/data_api/chain.rs

```diff
@@ -209,12 +213,45 @@ pub trait BlockSource {
         &self,
         from_height: Option<BlockHeight>,
         limit: Option<usize>,
-        with_row: F,
-    ) -> Result<(), error::Error<WalletErrT, Self::Error>>
+        with_block: F,
+    ) -> Result<Option<Range<BlockHeight>>, error::Error<WalletErrT, Self::Error>>
```

Comment:
Change needs to be documented in the changelog.

Reply:
Change was reverted.